spaces/S000192/properties/P000016.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000192
+property: P000016
+value: true
+---
+
+Given any open cover, the neighborhoods containing
+$\infty_E,\infty_D$ cover all but a bounded, and thus compact,
```

Comment:
could we say something slightly more precise like this maybe: ... the **elements of the cover** containing ... cover all but a **closed** bounded, ...